README.md

```diff
@@ -16,7 +16,7 @@ beautiful and functional.
 
 ## Consensus
 
-[`cumulus-consensus`](consensus) is a
+[`cumulus-consensus`](https://github.com/paritytech/cumulus/tree/master/client/consensus) is a
```

Comment:
Actually, shouldn't this be: https://github.com/paritytech/cumulus/blob/master/client/consensus/common/src/parachain_consensus.rs and renamed to `parachain-consensus` ?

Reply:
Sounds sensible.